packages/embeds/embed-react/vite.config.js

```diff
@@ -17,7 +17,7 @@ export default defineConfig({
     rollupOptions: {
       // make sure to externalize deps that shouldn't be bundled
       // into your library
-      external: ["react", "react-dom"],
+      external: ["react", "react/jsx-runtime", "react-dom", "react-dom/client"],
```

Comment:
We have react and react-dom as peer deps defined in package.json.

Do we not need to add react/jsx-runtime and react-dom/client?

Reply:
cc @KlotzJesse

Reply:
@hariombalhara hmm as far as i understand, as react/jsx-runtime is not a npm package but rather direct part of react (see node-modules/react), it will be sufficent enough to just mark it as external to stop it from being bundled